### DISCUSSION TITLE
<podcast:books>

### DISCUSSION THREAD

**Discussion by MerryOscar** [Podcastindex-org/podcast-namespace#555] [Tag Proposal]
I would like to propose a new `<podcast:books>` tag that would allow podcasters to reference books mentioned in their episodes in a structured way:

```xml
<podcast:books>
  <podcast:book title="" author="" url="" ibsn="" />
</podcast:books>
```

`<podcast:book />` attributes:
- **title** (*string*) - the title of the book 
- **author** (*string*) - the author of the book
- **url** (*string*) - a url linking to the book
- **isbn** (*string*) - the [International Standard Book Number (ISBN)](https://en.wikipedia.org/wiki/ISBN) for example `978-3-16-148410-0`

---

I see there being multiple benefits of referencing books this way as opposed to linking from the show notes:

- apps can display the information about the books in a structured way
- apps / services can provide a search capability for episodes that reference certain books
- podcasters using affiliate links can potentially make more income because the books are prominently surfaced

**Comment by ryan-lp** [comment]
You could use a URI/URN, which would also work more broadly than just books. For example:

```
urn:isbn:0-486-27557-4
```

Source: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#URLs_and_URNs

You can a separate lookup API to determine the author and other details from this.

**Comment by samsethi** [comment]
You could also reference an audiobook using the Remote Item Tag like a Podroll.

**Comment by theDanielJLewis** [comment]
How about making it media-agnostic, allowing a podcaster to use the same tag to recommend music, movies, TV shows, apps, physical products, and more?

There could be a `type` attribute to differentiate, and maybe some additional attributes for store-specific IDs. Like an Apple App Store ID, an Amazon US ASIN (and all the other locals), and so on.
